api/app.js

```diff
@@ -14,6 +15,15 @@ module.exports = {
         app.post('/sendCode', authController.sendCode);
         app.post('/sendcall', authController.sendCall);
         app.post('/signup', authController.signup);
+        app.post('/get-groups', groupAdminController.getGroups(mongoCollections));        
+        app.post('/get-dashboard-stats', groupAdminController.getDashboardStats(mongoCollections));
+        app.post('/get-messages-yearly-count', groupAdminController.getMessagesYearlyCount(mongoCollections));
+        app.post('/get-messages-monthly-count', groupAdminController.getMessagesMonthlyCount(mongoCollections));
+        app.post('/get-group-stats', groupAdminController.getGroupStats(mongoCollections));
+        app.post('/get-group-messages', groupAdminController.getGroupMessages(mongoCollections));
```

Comment:
I think that it will be much better if you will import specific collection inside specific controller instead of passing the whole database to each controller.